### PR TITLE
add transforms experimentation notebook 

### PR DESCRIPTION
**Pull request by jerryjliu**
notebook showing how you can use the ingestion pipeline to more easily experiment with different data parameters e.g. chunk size, metadata 

(not clear to me where this will live in the docs yet)

### CODE REVIEW

**Comment by logan-markewich** [comment]
I'm just noticing these docs aren't in the main TOC

https://github.com/run-llama/llama_index/blob/main/docs/module_guides/loading/ingestion_pipeline/root.md

File: docs/examples/transforms/TransformsEval.ipynb [hunk 1]
```diff
@@ -0,0 +1,1622 @@
+{
```
Comment (hatianzhang) [inline docs/examples/transforms/TransformsEval.ipynb:1]: <p>could we clean out those </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9005/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (hatianzhang) [inline docs/examples/transforms/TransformsEval.ipynb:1]: <p>any idea why summary is worse than default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9005/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (jerryjliu) [reply docs/examples/transforms/TransformsEval.ipynb:1]: it's possible there's a bug

Comment (jerryjliu) [reply docs/examples/transforms/TransformsEval.ipynb:1]: investigating right now

**Comment by jerryjliu** [comment]
> I'm just noticing these docs aren't in the main TOC
> 
> https://github.com/run-llama/llama_index/blob/main/docs/module_guides/loading/ingestion_pipeline/root.md

will add

File: docs/examples/transforms/TransformsEval.ipynb (hunk 1 above)
Comment (jerryjliu) [reply docs/examples/transforms/TransformsEval.ipynb:1]: actually did a quick check, seems fine. my hypothesis: 
- i wouldn't overfit to this dataset, it's quite toy
- extracting summary and putting in text can bias embeddings for retrieval or confuse llm with more context

**Pull request merged**